src/middleware/navigationMiddleware.ts

```diff
@@ -0,0 +1,62 @@
+import { NextFunction, Request, Response } from "express";
+import * as constants from "../lib/constants";
+import * as url from "node:url";
+import logger from "../lib/Logger";
+import { Navigation } from "types/navigation";
+
+export const NAVIGATION: Navigation = {
+    [constants.CHECK_MEMBER_DETAILS_FULL_URL]: {
+        allowedReferers: [constants.ADD_USER_FULL_URL, constants.CHECK_MEMBER_DETAILS_FULL_URL, constants.CANNOT_ADD_USER_FULL_URL],
+        redirectTo: constants.MANAGE_USER_FULL_URL
+    },
+    [constants.CONFIRMATION_MEMBER_ADDED_FULL_URL]: {
+        allowedReferers: [constants.CHECK_MEMBER_DETAILS_FULL_URL, constants.CONFIRMATION_MEMBER_ADDED_FULL_URL],
+        redirectTo: constants.MANAGE_USER_FULL_URL
+    },
+    [constants.getRemoveMemberCheckDetailsFullUrl("")]: {
```

Comment:
There should be no functions in the constants file. Please move getRemoveMemberCheckDetailsFullUrl() function to utils.

Reply:
Hi Maciej, I have resolved it, please double check again.